Date: Oct 8, 2018 7:39 PM
From: Kulbhushan Arora <Kulbhushan_Arora@comcast.com>
Subject: [PATCH] RDKALL-1221: Create separate CA bundles for browser apps
	as below removed cacert.pem as redundant heavy dust
        install browser based certs as per definition for browser
        based apps separately.
Source: COMCAST 02060561db73c28bcb9dff8d241c2275c6ad9b47 Mar 22, 2019 2:49 PM
Signed-off-by: Kulbhushan Arora <Kulbhushan_Arora@comcast.com>
Index: git/sbin/update-ca-certificates
===================================================================
--- git.orig/sbin/update-ca-certificates
+++ git/sbin/update-ca-certificates
@@ -26,6 +26,7 @@ fresh=0
 default=0
 CERTSCONF=$SYSROOT/etc/ca-certificates.conf
 CERTSDIR=$SYSROOT/usr/share/ca-certificates
+CERTSDIRBUNDLE=$SYSROOT/usr/share/ca-certificates/ca-certificates.crt
 LOCALCERTSDIR=$SYSROOT/usr/local/share/ca-certificates
 CERTBUNDLE=ca-certificates.crt
 ETCCERTSDIR=$SYSROOT/etc/ssl/certs
@@ -226,9 +227,10 @@ fi
 if [ -f "$TEMPBUNDLE" ]
 then
   chmod 0644 "$TEMPBUNDLE"
-  mv -f "$TEMPBUNDLE" "$CERTBUNDLE"
+  mv -f "$TEMPBUNDLE" "$CERTSDIRBUNDLE"
+  ln -sf "${CERTSDIRBUNDLE##$SYSROOT}" "$CERTBUNDLE"
   # Restore proper SELinux label after moving the file
-  [ -x /sbin/restorecon ] && /sbin/restorecon "$CERTBUNDLE" >/dev/null 2>&1
+  [ -x /sbin/restorecon ] && /sbin/restorecon "$CERTSDIRBUNDLE" >/dev/null 2>&1
 fi
 
 echo "$ADDED_CNT added, $REMOVED_CNT removed; done."
